druid/src/data.rs

```diff
@@ -269,6 +294,89 @@ impl<T0: Data, T1: Data, T2: Data, T3: Data, T4: Data, T5: Data> Data for (T0, T
     }
 }
 
+impl<T: 'static + ?Sized> Data for *const T {
+    fn same(&self, other: &Self) -> bool {
+        // This Does not dereference the pointer, only the reference to the pointer.
+        ptr::eq(*self, *other)
+    }
+}
+
+impl<T: 'static + ?Sized> Data for *mut T {
+    fn same(&self, other: &Self) -> bool {
+        // This Does not dereference the pointer, only the reference to the pointer.
+        ptr::eq(*self, *other)
+    }
+}
```

Comment:
I would leave these ones. When someone has raw pointers, they are better using `same_fn`. But if you add these, then also add `NonNull`

Reply:
Hmm OK I'm sold on removing the raw pointers.

Reply:
yes I agree I struggle to think of a good case where you'd want this.